content/download-self.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+// preventing reinjection by tabs.executeScript, just in case
+typeof self.oldCode !== 'string' && // eslint-disable-line no-unused-expressions
```

Comment:
We don't have to. You're adding two levels of nesting for what? Only to avoid using eslint comment? I'd rather disable this nonsensical rule in eslint config.

Reply:
I've succumbed to one `if`.